src/symm.cc

```diff
@@ -346,7 +359,7 @@ void symm(
                     Arow_0.releaseLocalWorkspace();
 
                     std::set<ij_tuple> tile_set;
-                    for (int64_t i = 0; i < A.mt(); ++i) {
+                    for (int64_t i = 1; i < A.mt(); ++i) {
```

Comment:
Again, seems like using i = 0 should not cause any problems and may be more conservative.